Library/Homebrew/cask/artifact/abstract_artifact.rb

```diff
@@ -129,9 +129,9 @@ def self.read_script_arguments(arguments, stanza, default_arguments = {}, overri
 
       attr_reader :cask
 
-      def initialize(cask, *dsl_args)
+      def initialize(cask, *args, **kwargs)
         @cask = cask
-        @dsl_args = dsl_args.deep_dup
+        @dsl_args = args.deep_dup + [kwargs.deep_dup]
```

Comment:
```suggestion
        @dsl_args = [*args.deep_dup, *kwargs.deep_dup].compact 
```

or something?

Feels like it might be nice to make these arguments explicit instead at some point so we can benefit from Sorbet

Reply:
Read this one wrong - this doesn't actually need changes.